.github/workflows/release-codedeploy-dev.yml

```diff
@@ -177,7 +177,6 @@ jobs:
 
   release:
     name: Release dev
-    if: ${{ github.ref_name == 'master' }}
```

Comment:
is this intentional?

Reply:
for temporarily allowing releasing from a feature branch. Re-enabling branch protection before merge.